cli/shared/src/main/scala/com/lightbend/rp/reactivecli/docker/CmdConfig.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.rp.reactivecli.docker
+
+import argonaut.CodecJson
+import com.lightbend.rp.reactivecli.docker.{ Config => RegistryConfig }
+
+case class CmdConfig(Config: RegistryConfig.Cfg) {
+  def registryConfig: RegistryConfig = RegistryConfig(Config)
+}
+
+object CmdConfig {
```

Comment:
Probably better to limit scope of this inside a helper object for invoking docker. Took me a minute to figure out what's actually happening here as the name doesn't tell much!

Reply:
I followed the pattern set by `SocketConfig`, which might have the same issue. Agreed I'll restrict the scope.